web-common/src/features/dashboards/leaderboard/Leaderboard.svelte

```diff
@@ -70,19 +74,58 @@
     runtime,
   } = getStateManagers();
 
+  const timeControlsStore = useTimeControlStore(StateManagers);
+
+  $: ({ instanceId } = $runtime);
+
   $: dimension = $getDimensionByName(dimensionName);
 
+  $: timeControls = $timeControlsStore;
+
   $: sortedQuery = createQueryServiceMetricsViewAggregation(
-    $runtime.instanceId,
+    instanceId,
     $metricsViewName,
     $leaderboardSortedQueryBody(dimensionName),
     $leaderboardSortedQueryOptions(dimensionName, visible),
   );
 
+  $: belowTheFoldDataQuery = createQueryServiceMetricsViewAggregation(
+    instanceId,
+    $metricsViewName,
+    {
+      dimensions: [{ name: dimensionName }],
+      whereSql: selectedBelowTheFold
```

Comment:
Would be better to use Expression while also merging with the general filters. Adding syntax mapping for things like measure filters would be redundant since backend does it from Expression object already.

Note that right now filters from other dimensions do not apply to this. You need to call `mergeMeasureFilters` and add this filter to the final filters.